contrib/brip/pybrip.h

```diff
@@ -0,0 +1,12 @@
+#ifndef pybrip_h_included_
+#define pybrip_h_included_
+
+#include <pybind11/pybind11.h>
+
+namespace pyvxl { namespace brip {
+
+void wrap_brip(pybind11::module &m);
```

Comment:
```suggestion
  void wrap_brip(pybind11::module &m);
```

Reply:
Should all the functions in the namespace be indented in the .cxx file as well?